vendor/github.com/robert-milan/go-object-interning/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Robert Milan And Mauro Stettler
```

Comment:
copyright should probably be assigned to grafana labs? but you can add yourselves to an `AUTHORS` file or something.